opengrep/vuln_python.py

```
@@ -0,0 +1,59 @@
+    conn = get_connection()
+    cursor = conn.cursor()
+    cursor.execute("SELECT id, email FROM users WHERE name = '" + name + "'")
+    rows = cursor.fetchall()
+    return {"results": rows}
+
+
+@app.route("/users/<int:user_id>")
+def get_user(user_id):
+    conn = get_connection()
+    cursor = conn.cursor()
+    cursor.execute(f"SELECT * FROM users WHERE id = {user_id}")
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)